### PR TITLE
159 get data from equations

### LINKED ISSUE #159: Retreive data from variables not graph
Since many get functions parse the graph and its nodes to get information which should be in the underlying internal structure - change the methods to use the internal structure.
See #156

### PR DESCRIPTION
**Pull request by skeating**
## Description
get_symbol_by_cmeta_id should be able to retrieve the symbol from equations without the graph.

## Motivation and Context
Fixes #159 
It is necessary for the unit conversion code I'm working on.
 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

NOTE: This technically is not a bug fix or new feature. It doesnt break functionality EXCEPT that getting a variable by cmeta_id, the variable will now no longer have the 'type' field set.

## Checklist:

- [x] I have updated all documentation necessary.
- [x] I have checked spelling in (new) comments.

## Testing
- [X] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically

### CODE REVIEW

**Comment by skeating** [comment]
This technically is not a bug fix or new feature. It doesn't break functionality EXCEPT that getting a variable by cmeta_id, the variable will now no longer have the 'type' field set (see lines 365-373 in test_model_functions.py).

File: tests/test_model_functions.py [hunk 1]
```diff
@@ -365,7 +365,12 @@ def test_remove_equation(self, local_hh_model):
         model = local_hh_model
         # Get model, assert that V is a state variable
         v = model.get_symbol_by_ontology_term(shared.OXMETA, 'membrane_voltage')
-        assert v.type == 'state'
+        # the issue here is that retrieving the variable uses the internal structure
```
Comment (MichaelClerx) [inline tests/test_model_functions.py]: Maybe reword a bit?

```
# Get the type from the variable graph
```
?
